docs/design/features/globalization-hybrid-mode.md

```diff
@@ -292,44 +292,118 @@ The number of `CompareOptions` and `NSStringCompareOptions` combinations are lim
 
 - All combinations that contain below `CompareOptions` always throw `PlatformNotSupportedException`:
 
-`IgnoreSymbols`,
+   `IgnoreSymbols`,
+
+   `IgnoreKanaType`,
+
+## String starts with / ends with
+
+Affected public APIs:
+- CompareInfo.IsPrefix
+- CompareInfo.IsSuffix
+- String.StartsWith
+- String.EndsWith
+
+Mapped to Apple Native API `compare:options:range:locale:`(https://developer.apple.com/documentation/foundation/nsstring/1414561-compare?language=objc)
+Apple Native API does not expose locale-sensitive endsWith/startsWith function. As a workaround, both strings get normalized and weightless characters are removed. Resulting strings are cut to the same length and comparison is performed. As we are normalizing strings to be able to cut them, we cannot calculate the match length on the original strings. Methods that calculate this information throw PlatformNotSupported exception:
+
+- [CompareInfo.IsPrefix](https://learn.microsoft.com/dotnet/api/system.globalization.compareinfo.isprefix)
+- [CompareInfo.IsSuffix](https://learn.microsoft.com/dotnet/api/system.globalization.compareinfo.issuffix)
+
+- `IgnoreSymbols`
+
+   As there is no IgnoreSymbols equivalent in NSStringCompareOptions all `CompareOptions` combinations that include `IgnoreSymbols` throw `PlatformNotSupportedException`
+
+## String indexing
+
+Affected public APIs:
+- CompareInfo.IndexOf
+- CompareInfo.LastIndexOf
+- String.IndexOf
+- String.LastIndexOf
+
+Mapped to Apple Native API `rangeOfString:options:range:locale:`(https://developer.apple.com/documentation/foundation/nsstring/1417348-rangeofstring?language=objc)
+
+In `rangeOfString:options:range:locale:` objects are compared by checking the Unicode canonical equivalence of their code point sequences.
+In cases where search string contains diaeresis and has different normalization form than in source string result can be incorrect.
+
+Here are the covered cases with diaeresis:
+  1. Search string contains diaeresis and has same normalization form as in source string.
+  2. Search string contains diaeresis but with source string they have same letters with different char lengths but substring is normalized in source.
+
+     a. search string `normalizing to form C` is substring of source string. example: search string: `U\u0308` source string:  `Source is \u00DC` => matchLength is 1
+
+     b. search string `normalizing to form D` is substring of source string. example: search string: `\u00FC` source string: `Source is \u0075\u0308` => matchLength is 2
+
+Not covered case:
+
+      Search string contains diaeresis and with source string they have same letters with different char lengths but substring is not
+      normalized in source. example: search string: `U\u0308 and \u00FC` source string: `Source is a\u0308\u0308a and \u0075\u0308`
+      as it is visible from example normalizaing search string to form C or D will not help to find substring in source string.
```

Comment:
I'm not sure if I'm understanding this. 
searchString `U\u0308 and \u00FC` (`Ü and ü`)
source string `a\u0308\u0308a and \u0075\u0308` (`ä̈a and ü`)
Is the search string supposed to be a substring in the source string? Was it supposed to be `U\u0308` and `\u00FC` should both be interpreted as substrings of `a\u0308\u0308a and (\u0075\u0308)` in that it matches the portion in the parenthesis?

I was actually expecting the not covered cases to be something like, source string is a sequence of unicode code points that represent a mixture of precomposed characters and decomposed characters.
i.e. Source string `H\u00EBll\u006F\u0308 World` (`Hëllö World`)
Search string `\u0065\u0308ll\u00F6` (`ëllö`)
These would bypass the precomposed vs precomposed comparison because it would be source 
string `H\u00EBll\u00F6 World` and search string `\u00EBll\u00F6`
However, neither the fully precomposed form of the search string `\u00EBll\u00FB` nor the fully decomposed form of the search string `\u0065\u0308ll\u006F\u0308` would match the unicode code point sequence in the source string `\u00EBll\u006F\u0308` so it would fall through the Form C and Form D fallbacks.

Reply:
My bad. Source should be `"Source is \u00DC and \u0075\u0308"`.
search string: `U\u0308 and \u00FC` (Ü and ü) source string: `Source is \u00DC and \u0075\u0308` (Source is Ü and ü). Will update it.